lib/marqeta/transaction.rb

```diff
@@ -60,5 +86,14 @@ def accessible_time_attributes
     def card_acceptor_hash
       attributes_hash['card_acceptor']
     end
+
+    def gateway_response_code
+      gpa_order = attributes_hash['gpa_order']
```

Comment:
If `gpa_order` is nil, what does that mean?

Reply:
@andyw8 There are cases where en error can happen before the 'order' is even placed. Eg: when a card is inactive. It will not have `gpa_order` then.

Reply:
Would it make sense to raise an exception if trying to access that when it isn't present?

Reply:
@andyw8 hmm, I think returning nil should be fine. Its a private method anyway. We want the user to be able to ask if it's `timeout?` without raising error.